rmw_zenoh_cpp/src/rmw_zenoh.cpp

```diff
@@ -2054,6 +2763,29 @@ rmw_wait(
     }
   }
 
+
+  if (services) {
```

Comment:
I wasn't able to fully grasp the idea of how waits and waits events are being used by `rcl` for the `rmw`. So the code is mere deducted from the other cases. In case you could provide more information here would be great for my understanding :smile: cc: @clalancette @Yadunund

Reply:
I think this is fine!